src/crypto/CryptoInterface.c

```diff
@@ -89,18 +106,19 @@ int BSL_Crypto_ClearKeyHandle(void *keyhandle)
 {
     CHK_ARG_NONNULL(keyhandle);
 
-    BSLB_CryptoKey_t *key = (BSLB_CryptoKey_t *)keyhandle;
-    CHK_POSTCONDITION(BSL_SUCCESS == BSLB_CryptoKey_Deinit(key));
+    BSL_CryptoKey_t *key = (BSL_CryptoKey_t *)keyhandle;
+    CHK_POSTCONDITION(BSL_SUCCESS == BSL_CryptoKey_Deinit(key));
```

Comment:
This logic will leak the struct if deinit fails. Is it actually possible for deinit to fail? Can it just return `void`?